### PR TITLE
Created CheepDTO and made it usable across the program

### PR DESCRIPTION
**Pull request by annahj1006**

### CODE REVIEW

**Review by Lukan707 — CHANGES_REQUESTED** [review]
Nice work.

Since names are not unique, and we want to request cheeps by author, when clicking an author, how about we add the id to the DTO, so we can send it back we the get request from the client.

File: src/Chirp.Razor/CheepContext.cs [hunk 1]
```diff
@@ -61,6 +61,7 @@ public class Cheep
 
 public class CheepDTO
 {
+    public required int CheepId { get; set; }
```
Comment (Lukan707) [inline src/Chirp.Razor/CheepContext.cs]: We might want to rename this to AuthorId, since it is the id of the author and not the id of the actual cheep.

**Review by Lukan707 — APPROVED** [review]
Nice :)

File: src/Chirp.Razor/CheepContext.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class ChirpDBContext : DbContext
 
 
 
-    public ChirpDBContext()
+    public ChirpDBContext() //Finds the current db in temp folder
```
Comment (AlbinoLoaf) [inline src/Chirp.Razor/CheepContext.cs:16]: Place the comment above the line that it pertains to

File: src/Chirp.Razor/CheepRepository.cs [hunk 3]
```diff
@@ -32,26 +31,30 @@ public void AddAuthor(string name, string email)
...
             Where(author => author.AuthorId == authorId).First(),
                 Text = text,
                 TimeStamp = DateTime.Now });
     }
 
-    public List<CheepViewModel> GetCheeps(int? pageNum)
+    public List<CheepDTO> GetCheeps(int? pageNum)
     {
-        List<CheepViewModel> cheepsToReturn = new List<CheepViewModel>();
+        //Creates a list of max 32 CheepDTO sorted by recent cheep
```
Comment (AlbinoLoaf) [inline src/Chirp.Razor/CheepRepository.cs:44]: The Max is not made here reference the point of creation

**Review by SkarpKat — APPROVED** [review]
Nice

**Pull request merged**